Apps/Admin/Server/appsettings.json

```diff
@@ -61,13 +61,13 @@
         "Base": "'self'",
         "DefaultSource": "'self'",
         "ScriptSource": "'self' 'wasm-eval' 'unsafe-eval' 'sha512-uKQ39gEGiyUJl4AI6L+ekBdGKpGw4xJ55+xyJG7YFlJokPNYegn9KwQ3P8A7aFQAUtUsAQHep+d/lrGqrbPIDQ==' 'sha256-Tui7QoFlnLXkJCSl1/JvEZdIXTmBttnWNxzJpXomQjg=' 'sha256-vY0Cwh5hLv8sUQ61m3KQgfrqeAmcY7t6wJ2FAdYPNyU='",
-        "ConnectSource": "'self' file: data: blob: filesystem: https://spt.apps.gov.bc.ca/com.snowplowanalytics.snowplow/tp2 https://*.loginproxy.gov.bc.ca/",
+        "ConnectSource": "'self' file: data: blob: filesystem: https://spt.apps.gov.bc.ca/com.snowplowanalytics.snowplow/tp2 https://*.loginproxy.gov.bc.ca/ https://loginproxy.gov.bc.ca/",
```

Comment:
doesn't `https://*loginproxy.gov.bc.ca/` (without period) cover the pattern `https://loginproxy.gov.bc.ca/`?

Reply:
hold I see the problem with my questioned pattern, alright ignore this